### PR TITLE
SCWSFP-19: Add Quote Entity

### PR DESCRIPTION
**Pull request by JordanAlbayrak**
### Context
- Adding Quote Backend functionalities (CRU) to allow for future quote generating and editing based on client service request form.
### Changes
- Created and configured necessary classes for Quote  (Entity, Controller, Service, Repo)
- Service and Controller Tests

Jira link: https://champlainsaintlambert.atlassian.net/browse/SCWSFP-19

### CODE REVIEW

File: Calligraphy/Controllers/QuoteController.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        [Consumes(MediaTypeNames.Application.Json)]
+        public IActionResult Create([FromBody] QuoteEntity quote)
+        {
+            var result = _quoteService.Create(quote);
+            if (result)
+            {
+                return Ok(quote);
+            }
+            return BadRequest();
+        }
```
Comment (Trim0500) [inline Calligraphy/Controllers/QuoteController.cs:53]: If the generation of a quote is tied to the posting of a form, similar to how mailer has been implemented, can you explain why creating a quote needs a separate endpoint? The quote generation and initial values are influenced by the form is it not?

**Review by Trim0500 — COMMENTED** [review]
I agree with most of the changes but you might want to take a look at my comments on the controller.

File: Calligraphy/Controllers/QuoteController.cs (hunk 1 above)
Comment (JordanAlbayrak) [reply Calligraphy/Controllers/QuoteController.cs:53]: I see your point, thanks!

Comment (JordanAlbayrak) [reply Calligraphy/Controllers/QuoteController.cs]: I fixed this issue and am now re-issuing this PR

**Review by Jaylapuck — APPROVED** [review]
LGTM

**Pull request merged**